src/elements/content-preview/PreviewLoading.scss

```diff
@@ -1,14 +1,15 @@
 @import '../common/variables';
 
-.bcpr {
-    .bcpr-loading {
-        color: $twos;
-        position: relative;
-        text-align: center;
-        z-index: 1;
+.bcpr-PreviewLoading {
+    align-items: center;
+    color: $bdl-gray;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
```

Comment:
Is `text-align` required if `align-items` is already present?

Reply:
Yup, since flexing is happening on the container that holds the text while text itself needs to have its own central align. Otherwise it will become paragraph style.